dictionaries/en_US/src/acronyms.txt

```diff
@@ -8,6 +9,8 @@ FLOTUS
 FOMO
 IFTTT
 IMAX
+lidar
+lidars
```

Comment:
I'm still confused why lidar is only in en_US and not in en_shared

Reply:
I found a reply here thanks

> The words `lidar` and `lidars` should have been added to `src/acronyms.txt`. > Better yet, an international English dictionary, but that doesn't exist yet.
> I'm moving `en_US.txt` under `src/legacy` to dissuade new additions
>
_Originally posted by @Jason3S in https://github.com/streetsidesoftware/cspell-dicts/pull/4325#issuecomment-2844234077_

Reply:
> Better yet, an international English dictionary, but that doesn't exist yet

But wait … isn't what `en_shared` is about?

Reply:
There isn't an `acronyms.txt` file there yet.

Reply:
Done with

- https://github.com/streetsidesoftware/cspell-dicts/pull/4340